requirements.in

```diff
@@ -0,0 +1,24 @@
+alembic
+arrow
+bleach
+cached-property
+docutils
+Flask==0.10.1
```

Comment:
Is there a reason this requirement is pinned? I'm not against pinning it in this file, just curious.